blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/BasicMetricsRWIntegrationTest.java

```diff
@@ -38,15 +37,13 @@
     private static final String TENANT2 = "987654";
     private static final String TENANT3 = "123789";
 
-    protected LocatorIO locatorIO = new DLocatorIO();
-    protected DelayedLocatorIO delayedLocatorIO = new DDelayedLocatorIO();
+    protected DLocatorIO locatorIO = new DLocatorIO();
```

Comment:
Not a big fan of this change. Cant you cast `locatorIO` to `DLocatorIO` where ever you need to call `getBoundStatementForLocator`? Now that I think about it, I dont know which one is less bad. Nevermind then.